src/equiv.js

```diff
@@ -126,6 +126,48 @@ QUnit.equiv = (function() {
 					parentsB.pop();
 					return true;
 				},
+				
+				"set": function( b, a ) {
+					var aArray, bArray;
+					
+					// b could be an object literal here
```

Comment:
On further inspection and tracing of the code, I believe it's impossible that a Set and a "non-Set" ends up in the switch case, as `a` is guaranteed to be same type as `b` (condition on line 245 of equiv.js). This begs the question: why does the "array" case include this check?